frontend/src/Settings/General/HostSettings.js

```diff
@@ -87,7 +88,7 @@ function HostSettings(props) {
       </FormGroup>
 
       {
-        enableSsl.value ?
+        enableSsl.value &&
```

Comment:
Changing this will cause enableSsl.value to render as well as the component below instead of just the component rendering, right?